test/events/test_send_update_events.py

```diff
@@ -52,6 +52,28 @@ def test_it_will_send_event_for_affiliation_insert(mock_publisher: MagicMock, pr
     assert mock_publisher.publish.call_args[0][0] == {'id': '12345678', 'type': 'profile'}
 
 
+def test_it_will_send_event_for_affiliation_deletion(mock_publisher: MagicMock, profile: Profile,
+                                                     session: scoped_session):
+    affiliation = Affiliation('1', Address(countries.get('gb'), 'City'), 'Organisation', Date(2017))
+
+    profile.add_affiliation(affiliation)
+    session.add(profile)
+
+    with patch('profiles.orcid.request'):
```

Comment:
two lines that need to be extracted given all the tests use them, and the `patch` distracts from the intent of the test

Reply:
You are right, this is what I ended up doing in the api tests:
https://github.com/elifesciences/profiles/blob/3e9d901b1fd208acc02e9d782fc3be2405aae114/test/test_api.py#L11

Needs to be available as a fixture now that it has use in other files.